### PR TITLE
Create CASDA astroquery module

### PR DESCRIPTION
**Pull request by jd-au**
A new astroquery module to allow ASKAP data to be queried by position.
Issue #1493

### CODE REVIEW

File: astroquery/casda/core.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            Display VOTable warnings or not.
+
+        Returns
+        -------
+        response : `requests.Response`
+            The HTTP response returned from the service.
+            All async methods should return the raw HTTP response.
+        """
+        request_payload = self._args_to_payload(coordinates=coordinates, radius=radius, height=height,
+                                                width=width, kwargs=kwargs)
```
Comment (bsipocz) [inline astroquery/casda/core.py]: kwargs is not used for anything, remove it

File: astroquery/casda/core.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    URL = conf.server
+    TIMEOUT = conf.timeout
+
+    # all query methods are implemented with an "async" method that handles
+    # making the actual HTTP request and returns the raw HTTP response, which
+    # should be parsed by a separate _parse_result method.   The query_object
+    # method is created by async_to_sync automatically.
+    def query_region_async(self, coordinates, radius=None, height=None, width=None,
+                           get_query_payload=False, cache=True,
+                           **kwargs):
```
Comment (bsipocz) [inline astroquery/casda/core.py]: kwargs is not used later on, please remove and add `verbose` directly if you plan to use it, or remove it, too from the parameters list from below, if it's not used.

**Comment by bsipocz** [comment]
@jd-au - the missing remote tests are the real blockers atm. They are not run from CI all the time, but we need them to make sure the code is not broken due to changes on the server etc.

**Comment by bsipocz** [comment]
@jd-au - This all looks good now, thank you. One last thing, would you mind rebasing to we get rid of the merge commits? alternatively I'm happy to do it before merging.

**Comment by jd-au** [comment]
@bsipocz and @keflavich Thank-you for the review. I've completed the rebase and corrected the version in changes.rst so I think this is ready to go, pending the travis build passing.

**Comment by bsipocz** [comment]
Thank you @jd-au!

If everything goes well, give it a few hours and installing astroquery with 
``pip install --pre astroquery`` would include CASDA 🎉

**Pull request merged**